src/test/scala/outwatch/OutWatchDomSpec.scala

```diff
@@ -824,6 +824,29 @@ class OutWatchDomSpec extends JSDomSpec {
     node.disabled shouldBe true
   }
 
+  it should "correctly work with AsVDomModifier conversions" in {
+
+    val node = document.createElement("div")
+
+    OutWatch.renderReplace(node, div("one")).unsafeRunSync()
+    node.innerHTML shouldBe "one"
+
+    OutWatch.renderReplace(node, div(1)).unsafeRunSync()
+    node.innerHTML shouldBe "1"
+
+    OutWatch.renderReplace(node, div(1.0)).unsafeRunSync()
+    node.innerHTML shouldBe "1"
```

Comment:
Why is a `Double` `1.0` rendered as `1` and not `1.0`?

Reply:
Apparently that's how Scala.js renders `Double.toString`:

https://scalafiddle.io/sf/OcJkymj/0

Reply:
Ok! I just tried in the scala REPL and it was "1.0".

Reply:
Scala REPL is JVM.

Reply:
I wanted to say that. 😄